bench-routes.service

```diff
@@ -0,0 +1,12 @@
+#Service script for bench-route
+
+[Unit]
+Description= Systemd bench-routes service
+After=network-online.target network.target
+Requires=network-online.target network.target
+
+[Service]
+Type=simple
+RemainAfterExit=yes
+ExecStart=/home/ps/bench-routes 9090 start > /dev/null  2>&1
```

Comment:
Please change this to the `bench-routes` since after placing in the `bin`, this can be directly accessed

Reply:
thanks for reviewing.
My comments:
1) We can get rid of 'stop' & hence get rid of ExecStop.Because the default behavior of ExecStop is to kill the process. (at the end that's what we want to achieve)
2) The reason why I silenced the command o/p with /dev/null 2>&1 is to make sure we don't flood the systemd logs. 
3) I think we should provide absolute path of the bench-routes binary. I have seen systems that don't include /bin OR /usr/bin in PATH variable. Providing an absolute path will always give the location of binary.